.github/workflows/integration-test.yaml

```diff
@@ -5,21 +5,66 @@ on:
 
 jobs:
   integration-test:
-    runs-on: ubuntu-22.04
+    runs-on: sd-core-runner
```

Comment:
Without cos it should be ok, but then again, the `sdcore` bundle will never go fully `Active-Idle`. That's because of the `grafana-agent` which requires Prometheus.